.gitignore

```diff
@@ -54,3 +54,6 @@ node_modules
 # Lighthouse
 .lighthouseci
 lighthouse-results.md
+
+# Yarn Plug'n'Play (PnP) files
```

Comment:
Was this change intentional? I think we want to keep the `.pnp.*` files committed. Otherwise, devs will have to remember to run `yarn` any time `package.json` or `yarn.lock` changes.

Reply:
It appears that the yarn developers intend for PnP files to be committed when using zero-install mode, which we are.

https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Reply:
Yes, as the pnp files would always be edited after I ran `yarn` (without me making any changes). This causes lots of issues when checking out different branches.

I'm happy to keep them on this repo, but I need to find a way to ensure they don't change after running `yarn`. I'll take a look

Reply:
In the past, I've seen running `yarn` change the `@surma/rollup-plugin-off-main-thread` entry in `.pnp.cjs`. I thought you fixed that in d8180b5c0bb173794ee4c67a1cadfa921a55a0c3 though. What changes is `yarn` making to `.pnp.cjs` when you switch branches?